x/interchainstaking/keeper/receipt.go

```diff
@@ -203,7 +168,35 @@ func (k *Keeper) MintAndSendQAsset(ctx sdk.Context, sender sdk.AccAddress, sende
 
 	switch {
 	case zone.ReturnToSender || memoRTS:
-		err = k.SendTokenIBC(ctx, k.AccountKeeper.GetModuleAddress(types.ModuleName), senderAddress, zone, qAssets[0])
```

Comment:
Any reason why the method was removed here?

Reply:
this is old version, I reverted it